packages/data-stores/src/plans-ui/index.ts

```diff
@@ -0,0 +1,30 @@
+import { registerStore } from '@wordpress/data';
+import { controls } from '@wordpress/data-controls';
+import * as actions from './actions';
+import { STORE_KEY } from './constants';
+import reducer, { State } from './reducer';
+import * as selectors from './selectors';
+import type { SelectFromMap, DispatchFromMap } from '../mapped-types';
+
+export type { State };
+
+let isRegistered = false;
+
+export function register(): typeof STORE_KEY {
+	if ( ! isRegistered ) {
+		isRegistered = true;
+		registerStore< State >( STORE_KEY, {
+			actions,
+			controls,
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+			reducer: reducer as any,
```

Comment:
I know it's a pervasive pattern, but this can be 
`reducer: reducer as Reducer< State, AnyAction >` to avoid disabling eslint, or be even more specific as
`reducer: reducer as Reducer< State, WpcomPlansUIAction >`

Reply:
good call